### DISCUSSION TITLE
Does the github education for student  conflict with the github copilot 30 day trial?

### DISCUSSION THREAD

**Discussion by wheeleryanli** [community/community#138350] [GitHub Education]
### Hi everyone,

I plan to use github copilot to help me program, my github education has been approved, but my benefits are not currently available, I am going to choose to try it for 30 days first, will this action conflict with my benefits, and will my trying it out result in my benefits not arriving to take effect?

**Comment by jingbof** [comment]
Your GitHub Education benefits, including access to GitHub Copilot, typically arrive within 72 hours of your application being approved. (However, during busy periods like the start of a new semester, this process may take longer)

And no, starting a 30-day trial for GitHub Copilot should not conflict with your GitHub Education benefits. Typically, the trial period is separate from any benefits provided through programs like GitHub Education. Once the trial ends, your education benefits should automatically apply if they have been approved and are available.

Or if you can wait a few more days, your education benefits may activate soon, giving you access to GitHub Copilot without needing the trial.

**Comment by pg56714** [comment]
@wheeleryanli 
Mark me as answer thanks

[copilot](https://github.com/features/copilot)

Click **Get started with Copilot**.

Click **Start a free trial**.

That's awesome! It really is free to use, with the big text saying, **Congratulations! You are eligible to use GitHub Copilot for free**.

![image](https://github.com/user-attachments/assets/3db0309c-4888-435a-9378-e57d850093c2)
![image](https://github.com/user-attachments/assets/bb49cfe9-4cde-460e-9503-a38b28fb3648)
![image](https://github.com/user-attachments/assets/d3fa07f3-d37f-4561-9d40-39c0f879fdf7)
